src/OrchardCore.Modules/OrchardCore.Flows/wwwroot/Scripts/flows.edit.js

```diff
@@ -1,23 +1,26 @@
-var createEditorUrl = $('#buildEditorUrl').attr("value");
-var widgetTemplate = function (data, prefixesName, prefix, contentTypesName, contentType) {
-    return '<div class="widget-template col-md-12">' + data + '<input type="hidden" name="' + prefixesName + '" value="' + prefix + '" /><input type="hidden" name="' + contentTypesName + '" value="' + contentType + '" /></div>';
-};
 //variables used in FlowPart.Edit sortable
 var widgetDragItem, lastContainer, widgetItemSourceId, widgetItemDestId;
 
-function guid() {
-    function s4() {
-        return Math
-            .floor((1 + Math.random()) * 0x10000)
-            .toString(16)
-            .substring(1);
-    }
-    return s4() + s4() + s4() + s4() + s4() + s4() + s4() + s4();
-}
 
 $(function () {
 
+    // function scopped variable    
+    var widgetTemplate = function (data, prefixesName, prefix, contentTypesName, contentType) {
+        return '<div class="widget-template col-md-12">' + data + '<input type="hidden" name="' + prefixesName + '" value="' + prefix + '" /><input type="hidden" name="' + contentTypesName + '" value="' + contentType + '" /></div>';
+    };
+
+    function guid() {
+        function s4() {
+            return Math
+                .floor((1 + Math.random()) * 0x10000)
+                .toString(16)
+                .substring(1);
+        }
+        return s4() + s4() + s4() + s4() + s4() + s4() + s4() + s4();
```

Comment:
Use power or
```javascript
return 8 * s4();
```

Reply:
I din't change the exiting function, just moved inside $() scope,  I have no intention to change the log of the code